example/deploy/csi-mock-driver-deployment.yaml

```diff
@@ -0,0 +1,125 @@
+# external-resizer and mock-driver
+#
+# This YAML file also includes deployment of the CSIDriver, StorageClass
+# VolumeSnapshotClass for the mock driver.
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-mockplugin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-mockplugin
+  template:
+    metadata:
+      labels:
+        app: csi-mockplugin
+    spec:
+      serviceAccount: csi-mock
+      containers:
+        - name: csi-provisioner
+          image: gcr.io/k8s-staging-csi/csi-provisioner:canary
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: driver-registrar
+          image: gcr.io/k8s-staging-csi/csi-node-driver-registrar:canary
+          args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/io.kubernetes.storage.mock/csi.sock
+          env:
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+          - mountPath: /registration
+            name: registration-dir
+        - name: csi-resizer
+          image: gcr.io/k8s-staging-csi/csi-resizer:canary
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+        - name: csi-snapshotter
+          image:  gcr.io/k8s-staging-csi/csi-snapshotter:canary
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election=false"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: mock-driver
+          image: gcr.io/k8s-staging-csi/mock-driver:canary
+          args:
+            - "--disable-attach=true"
+          env:
+            - name: CSI_ENDPOINT
+              value: /csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+      volumes:
+        - name: socket-dir
+          hostPath:
+            path: /var/lib/kubelet/plugins/io.kubernetes.storage.mock
+            type: DirectoryOrCreate
+        - name: registration-dir
+          hostPath:
+            path: /var/lib/kubelet/plugins_registry
+            type: Directory
+
+---
+apiVersion: storage.k8s.io/v1
```

Comment:
v1 was made available in 1.18. So we should document that this example requires 1.18+

Reply:
Done